tests/end2end/playwright/edition-form.spec.js

```diff
@@ -0,0 +1,114 @@
+      await expect(page.locator('#jforms_view_edition input[name="integer_field"]')).toHaveAttribute('step','5');
+      await expect(page.locator('#jforms_view_edition input[name="integer_field"]')).toHaveAttribute('min','-200');
+      await expect(page.locator('#jforms_view_edition input[name="integer_field"]')).toHaveAttribute('max','200');
+
+      // add data
+      await page.locator('#jforms_view_edition input[name="integer_field"]').fill('50');
+
+      // submit form
+      await page.locator('#jforms_view_edition__submit_submit').click();
+      // will close & show message
+      await expect(page.locator('#edition-form-container')).toBeHidden();
+      await expect(page.locator('#lizmap-edition-message')).toBeVisible();
+  })
+
+  test('Boolean nullable w/ value map', async ({ page }) => {
+
+      let editFeatureRequestPromise = page.waitForResponse(response => response.url().includes('editFeature'));
+
+      await page.locator('#button-edition').click();
+      await page.locator('#edition-layer').selectOption({label: 'many_bool_formats'});
+      await page.locator('#edition-draw').click();
+      await page.locator('#jforms_view_edition_liz_future_action').selectOption('edit');
+      await page.getByLabel('bool_simple_null_vm').selectOption('t');
+      await page.locator('#jforms_view_edition__submit_submit').click();
+
+      await editFeatureRequestPromise;
+
+      // Wait a bit for the UI to refresh
+      await page.waitForTimeout(300);
+
+      await expect(page.getByLabel('bool_simple_null_vm')).toHaveValue('t');
+
+      await page.getByLabel('bool_simple_null_vm').selectOption('');
+      await page.locator('#jforms_view_edition__submit_submit').click();
+
+      await editFeatureRequestPromise;
+
+      // Wait a bit for the UI to refresh
+      await page.waitForTimeout(300);
+
+      await expect(page.getByLabel('bool_simple_null_vm')).toHaveValue('');
+  })
+
+  test('Error fetch form', async ({ page }) => {
+      await page.route('**/edition/createFeature*', async route => {
+          await route.fulfill({
+              status: 404,
+              contentType: 'text/plain',
+              body: 'Not Found!'
+          });
+      });
+
+      // display form
+      await page.locator('#button-edition').click();
+      await page.locator('a#edition-draw').click();
+
+      // message
+      await expect(page.locator("#lizmap-edition-message")).toBeVisible();
+      await expect(page.locator("#message > div")).toHaveClass(/alert-error/);
+  })
+
+  test('Error send feature', async ({ page }) => {
+      // display form
+      await page.locator('#button-edition').click();
+      await page.locator('a#edition-draw').click();
+
+      // add data
+      await page.locator('#jforms_view_edition input[name="integer_field"]').fill('50');
+
+      await page.route('**/edition/saveFeature*', async route => {
+          await route.fulfill({
+              status: 404,
```

Comment:
If it works for 50X error as well, LGTM ;-)

Thanks a lot

This is only client side. For PHP side, we must check that the 500 error is correctly logged in the correct log file for GIS administrator.